rts/lra/lra-test/lra-arquillian-extension/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all directories in the `lra-test` folder is named without the `lra-` prefix. Again it's a bit picky but for me it's nicer to be consistent. Or let me know if you don't agree :)